src/main/java/org/folio/marc/migrations/domain/entities/Operation.java

```diff
@@ -23,7 +26,10 @@
 @Setter
 @Entity
 @ToString
+@Builder(toBuilder = true)
+@NoArgsConstructor
 @Table(name = "operation")
+@AllArgsConstructor
```

Comment:
Why this Builder is needed there? Is it only for tests?

Reply:
yes, i used for testing mainly i thought maybe it would be useful to have it, but gonna remove it coz will get entityType from jobParameters